### ISSUE TITLE
vacuum hopper

### ISSUE THREAD

**Issue by niall4** [davqvist/Exoria#266]
Having problem with the vacuum hopper not letting mesh through into the chest. Item gets stuck in the internal buffer and you have to break item to get it out

**Comment by davqvist** [comment]
Stupid question: Are you sure the chest is not full?

**Comment by niall4** [comment]
Yeah positive I was using a double chest and it was less than half empty

**Comment by QUEWEX** [comment]
I had a similar issue with ember pipes and bees. I was using ember piping to extract from the first bee attractor multiblock into a chest (vanilla chest made with iron nuggets) but the pipe would only dump bees into the first two slots in the upper left of the chest, and anything else would get backed up in the pipe. If I moved the bees from those first two slots to another position in the chest, the pipe would dump two more bees then stop transferring again. Bees were my main experience with this problem but there was some other non-stackable item I had this issue with as well, though I don't remember what it was. (I didn't test other item mod item transferring methods, e.g. inserters or IE extractors, to see whether the issue might have been non-stacking items into chests in general.)

Transferring stackable items with embers worked fine. 

I presumed it is simply a bug or a mistake in how embers handles item transferring into chests (my experience was with pipes but it is not a stretch to think it would apply to vacuums as well). Given embers is largely out of development, I don't think it could be fixed.
